themes/bootstrap3/js/common.js

```diff
@@ -239,21 +239,52 @@ var VuFind = (function VuFind() {
     return html.replace(/(<script[^>]*) nonce=["'].*?["']/ig, '$1 nonce="' + getCspNonce() + '"');
   };
 
+  function setInnerHtml(elm, html) {
+    elm.innerHTML = html;
+    Array.from(elm.querySelectorAll("script")).forEach(oldScript => {
+      const newScript = document.createElement("script");
+      Array.from(oldScript.attributes).forEach(attr => {
```

Comment:
I don't think it's necessary to copy the attributes one by one. Actually, you could just copy the whole script as is and just replace the nonce attribute before replaceChild below. Also, the code needs to check script type before modifying it. We have some that are not JavaScript. See https://github.com/vufind-org/vufind/blob/cbd39fcd384bec72d9a798cb97a35567b2880438/themes/bootstrap3/js/search.js#L335 and https://github.com/vufind-org/vufind/blob/cbd39fcd384bec72d9a798cb97a35567b2880438/themes/bootstrap3/js/search.js#L351 for example. Perhaps we could make it into a shared helper method?